src/part02.rs

```diff
@@ -17,7 +17,7 @@ pub enum SomethingOrNothing<T>  {
 }
 // Instead of writing out all the variants, we can also just import them all at once.
 pub use self::SomethingOrNothing::*;
-//@ What this does is to define an entire family of types: We can now write
+//@ What this does is define an entire family of types: We can now write
```

Comment:
Are you a native speaker? I'm not, and I just want to be sure here ;-) . I thought one could do "is defining" and "is to define" here, but not "is define".